imhamt/src/hash.rs

```diff
@@ -37,7 +37,7 @@ pub struct LevelIndex(pub usize);
 
 impl LevelIndex {
     #[inline]
-    pub fn mask(&self) -> u32 {
+    pub const fn mask(&self) -> u32 {
         1u32.wrapping_shl(self.0 as u32)
```

Comment:
Is this meant to silently produce a zero bit mask for levels above 32? There is probably no other alternative if you want to have it as a const fn, just wanted to make sure.

Reply:
yes, it's fine in this case, it should never need more